ONSdigital/census-rm-case-processor
# Pseudorandom case ref for 2021

# Motivation and Context
Creating totally random caserefs will result in a 1 in 3 chance of a 'clash' with an existing clash for 2021, which will badly hurt performance. We need a better solution.

# What has changed
Use a pseudorandom number generator, so that the case refs are always unique, but appear to be random.

# How to test?
Run the unit test marked as `@Ignore`.

Run the acceptance tests to check there's no regression.

# Links
Trello: https://trello.com/c/e9UNd6NM
